web-wallet/src/routes/(welcome)/setup/restore/MnemonicAuthenticate.svelte

```diff
@@ -1,28 +1,35 @@
 <script>
-	import { Card, Mnemonic } from "$lib/dusk/components";
-	import { toast } from "$lib/dusk/components/Toast/store";
-	import { validateMnemonic, wordlists } from "bip39";
-	import { mdiAlertOutline } from "@mdi/js";
+  import { Card, Mnemonic } from "$lib/dusk/components";
+  import { toast } from "$lib/dusk/components/Toast/store";
+  import { validateMnemonic, wordlists } from "bip39";
+  import { mdiAlertOutline } from "@mdi/js";
 
-	/** @type {boolean} */
-	export let isValid = false;
+  /** @type {boolean} */
+  export let isValid = false;
 
-	/** @type {number} */
-	export let wordLimit = 12;
+  /** @type {number} */
+  export let wordLimit = 12;
 
-	/** @type {string[]} */
-	export let enteredMnemonicPhrase = [];
+  /** @type {string[]} */
+  export let enteredMnemonicPhrase = [];
 
-	$: isValid = validateMnemonic(enteredMnemonicPhrase.join(" "), wordlists.english);
-	$: {
-		if (enteredMnemonicPhrase.filter(word => word !== "").length === wordLimit && !isValid) {
-			toast("error", "Invalid mnemonic phrase", mdiAlertOutline);
-		}
-	}
+  $: isValid = validateMnemonic(
+    enteredMnemonicPhrase.join(" "),
+    wordlists.english
+  );
+  $: {
```

Comment:
I will open an issue for this one – the additional code block is not needed. `$: if...` should be enough.